README.md

```diff
@@ -28,6 +28,26 @@ remotes::install_github("NOAA-FIMS/FIMS")
 library(FIMS)
 ```
 
+#### Fixing Fatal Error
+
+Windows users can expect to see some derivative of the following error message in their R session if they have not yet set some flags using {withr}.
+```
+Fatal error: can't write <xxx> bytes to section .text of FIMS.o: 'file too big
+```
+You can easily fix this by running
```

Comment:
we should add "the following line in your local R session. Note that this call will need to be repeated each time you open a new session." Or, at least that is what it sounded like needed to happen when you were giving instructions to Cole.